### ISSUE TITLE
Chips with autocomplete (no jQuery and 1.0.0-alpha3) does not initialize

### ISSUE THREAD

**Issue by marcosdefontes** [Dogfalo/materialize#5515]
## Expected Behavior
Create the Chips component with the autocomplete option using Javascript (no jQuery).

## Current Behavior
Running the component I get the message
```
materialize.min.js:6 Uncaught TypeError: Cannot read property '0' of null
at n.value (https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0-alpha.3/js/materialize.min.js:6:104488)
    at new n (https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0-alpha.3/js/materialize.min.js:6:101364)
    at Function.value (https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0-alpha.3/js/materialize.min.js:6:11799)
    at Function.value (https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0-alpha.3/js/materialize.min.js:6:106244)
```
## Steps to Reproduce
Codepen link: [Using 1.0.0-alpha3](https://codepen.io/marcosdefontes/pen/VyzeNL)
HTML:
```html
<div class="row">
  <div class="col s12 m6">
    Groups:                            
    <div class="chips chips-autocomplete-groups">     </div>
  </div>                       
</div>
```
JS:
```javascript
var elem = document.querySelector('.chips-autocomplete-groups');
if (elem != null) {
    var insChipAC = M.Chips.init(elem, {
        placeholder: 'Enter a group',
        secondaryPlaceholder: '+Group',
        autocompleteOptions: {
            data: {
                'Alex': null,
                'Brandy': null,
                'Charlie': null
            },
            limit: Infinity,
            minLength: 1
        }        
    });    
}
```
## Context
In **1.0.0-alpha2** version I was able to create the component
Codepen link: [Using 1.0.0-alpha2](https://codepen.io/marcosdefontes/pen/XVadob)

## Your Environment

* Version used: 1.0.0-alpha3
* Browser Name and version: Chrome 63.0.3239.108
* Operating System and version (desktop or mobile): Ubuntu 16.04

**Comment by ntidbury** [comment]
Have the same issue, issue is also found in the Materialize Documentation.

**Comment by DanielRuf** [comment]
Did you try `new M.Chips(` instead?
PS: `elem.length` might ne a better check.

**Comment by marcosdefontes** [comment]
Yes, I tried.
Same error:
```
Uncaught TypeError: Cannot read property '0' of null
    at n.value (materialize.min.js:6)
    at new n (materialize.min.js:6)
    at pen.js:3
```
https://codepen.io/marcosdefontes/pen/LezPMZ

**Comment by Dogfalo** [comment]
Fixed by #5514

**Comment by firelizzard18** [comment]
A better solution is to fix `Component.init()` (which is what the above reference is about). That method will happily take a jquery result or a node list, but it barfs on cash results. Adding `|| !!els.cash` fixes the issue and preserves developer assumptions about how the method should work.
